package.json

```diff
@@ -67,8 +67,8 @@
     "d3": "3.5.5",
     "datatables.net": "1.10.10",
     "datatables.net-responsive": "2.0.1",
-    "fec-style": "~1.7",
-    "handlebars": "3.0.3",
+    "fec-style": "git+https://github.com/18f/fec-style#filter-data-update",
```

Comment:
Just need to update to point to next minor version, should be `~1.8` once we're ready with `fec-style`.